### PR TITLE
[MobSkills] Exuviation and Magma Fan

### PR DESCRIPTION
**Pull request by nesstea**
I don't know what the exact damage or heal values should be. These seem significant enough from my memory.

### CODE REVIEW

File: scripts/globals/mobskills/Magma_Fan.lua [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+	return 1;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+	local power = math.ceil(mob:getMainLvl() / 5);
```
Comment (bendangelo) [inline scripts/globals/mobskills/Magma_Fan.lua]: Power isn't used.

File: scripts/globals/mobskills/Exuviation.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    while (dispel ~= EFFECT_NONE)
+    do
+        effectCount = effectCount + 1;
+        dispel = mob:eraseStatusEffect();
+    end;
+
+    if (effectCount > 0) then
+        skill:setMsg(MSG_SELF_HEAL);
+        mob:addHP(heal * effectCount);
+        return (heal * effectCount);
```
Comment (bendangelo) [inline scripts/globals/mobskills/Exuviation.lua]: Please use `return MobHealMove(mob, heal * effectCount)`. This will prevent over healing.

**Comment by nesstea** [comment]
Amended commit @bendangelo

**Comment by bendangelo** [comment]
thanks

**Pull request merged**